examples_tests/41.VisibilityBuffer/shadeVBuffer.comp

```diff
@@ -174,6 +192,14 @@ void main()
         ViewPos = pos-pc.camPos; // weird global for MTL shader
         nbl_glsl_IsotropicViewSurfaceInteraction interaction = nbl_glsl_calcSurfaceInteraction(pc.camPos,pos,normal);
         color.rgb = nbl_computeLighting(interaction);
+
+        visibleBuff.visible[drawGUID] = uint16_t(1u);
+
+        if(gl_GlobalInvocationID.x == 0)
+        {
+            cubeIndirectDraw.draw.instanceCount = 0;
```

Comment:
`occlusionCullMap.comp` could reset the `cubeIndirectDraw`

Reply:
not really
https://github.com/Devsh-Graphics-Programming/Nabla/blob/przemog_occlusion_culling/examples_tests/41.VisibilityBuffer/occlusionCullMap.comp#L24

Reply:
ah yeah.